# msg/TrackingState.msg

sensor_msgs/Image image_annotated
sensor_msgs/Image image_roi
std_msgs/Bool tracking_stable
std_msgs/String pose_type
